atom/browser/api/atom_api_notification.cc

```diff
@@ -56,7 +56,8 @@ Notification::Notification(v8::Isolate* isolate,
                            mate::Arguments* args) {
   InitWith(isolate, wrapper);
 
-  presenter_ = brightray::BrowserClient::Get()->GetNotificationPresenter();
+  presenter_ = static_cast<AtomBrowserClient*>(AtomBrowserClient::Get())
```

Comment:
The only other reference to `brightray::BrowserClient` is in its creation with `brightray::BrowserMainDelegate`, can you move it to `atom` and remove `brightray::BrowserClient` completely.

Reply:
@deepak1556 I don't understand, can you be more specific?

Reply:
To remove this cast we need to remove `brightray::BrowserClient`, the last use of this class has been shifted out of `brightray` with this PR, so I am suggesting to remove `brightray/browser/browser_client.{cc|h}` alongside.

Reply:
Got it. I wanted to minimized the changes to just notifications in this PR. I can do that in a separate PR.

Reply:
Sure lets fix it as a follow up 👍